bigdeal.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
You probably want to full protection bash provides:

```suggestion
set -euo pipefail
```